xbmc/addons/kodi-dev-kit/include/kodi/addon-instance/Game.h

```diff
@@ -948,16 +948,45 @@ class ATTRIBUTE_HIDDEN CInstanceGame : public IAddonInstance
     return GAME_ERROR_NOT_IMPLEMENTED;
   }
 
+  //============================================================================
+  /// @brief Generates a hash from a file based on the console id
+  ///
+  /// @param[out] hash The hash of the file
```

Comment:
Can you be a bit more descriptive about what an rhash is and how it's derived? I think this information would be helpful.